ee/desktop/release-notes.md

```diff
@@ -15,6 +15,15 @@ For information on system requirements, installation, and download, see:
 
 For Docker Enterprise Engine release notes, see [Docker Engine release notes](/engine/release-notes).
 
+## Version 2.1.0.7
+2019-10-18
+
+Docker Desktop Enterprise 2.1.0.7 contains a Kubernetes upgrade. Note that your local Kubernetes cluster will be reset after installing this version.
```

Comment:
there's no Kube uprade in this patch release, so this can be removed

Reply:
Thanks. Fixed this.